src/components/dashboard/sidebar/utils.js

```diff
@@ -0,0 +1,6 @@
+export const getOfferExpiringFirst = (offers) => offers?.filter(offer => offer.isCurrent)
+  .sort((a, b) => new Date(a.endDatetime) - new Date(b.endDatetime))[0];
+
+export const getPolicyExpiringFirst = (policies) => policies?.sort(
+  (a, b) => new Date(a.subsidyExpirationDate) - new Date(b.subsidyExpirationDate),
```

Comment:
[clarification] Should we be checking only for non-expired, active policies here similar to `getOfferExpiringFirst`'s use of `offer.isCurrent`? 

Note: we probably don't need to if the `credits_available` API is already only returning active/current offers, which it seems like it might be? Just a sanity check 😄